auctions/views.py

```diff
@@ -17,17 +19,59 @@ class BidStatusView(LoginRequiredMixin, TemplateView):
     """
     template_name = "addon/widget.html"
 
-    def get_context_data(self, **kwargs):
-        url = self.request.GET['url']
+    def _get_bid_and_claim(self, url):
         bid = None
         claim = None
         try:
             bid = Bid.objects.get(user=self.request.user, url=url)
             claim = Claim.objects.get(issue=bid.issue)
         except:
             pass
```

Comment:
At some point we need to remove all these `except: pass` and `except: return False` things. We should allow the exceptions to be raised to the user, and capture/log them in New Relic.